.changes/next-release/bugfix-6ce9e38b-9d86-4273-a47a-34595c4ef2a1.json

```
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix for AmazonQ on Linux input focus problem https://github.com/aws/aws-toolkit-jetbrains/issues/4100"
```

```suggestion
  "description" : "Fix for AmazonQ on Linux input focus problem (#4100)"
```
automation automatically turns this into links

---

Done